Chelakhovl/HW7/task3.py

```diff
@@ -0,0 +1,10 @@
+def calculates_letters(word):
```

Comment:
good job
look at this version of code 
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result